testCases/testShapeGroup.js

```diff
@@ -66,7 +66,47 @@ const data: SJShapeGroupLayer = {
         "patternTileScale": 1
       }
     ],
+    "innerShadows": [{
```

Comment:
how did you arrive at this `json` based on the existing group test case? I was expecting a separate test case directly copy-pasted out of sketch. If this parses fine, I don't have an issue w/ it but my idea with the test cases is to ensure we're testing against actual sketch output.

Reply:
I took the output of a sketch file with a shape with innerShadow and copied it over. So it is actual sketch output. I'm happy to create an entirely new file. But since shadows are tested along with the textShapeGroup.js I thought it might be a good idea to also test the innerShadows there as well. Up to you, just let me know. I'm totally fine to create a new file.